motor/core.py

```diff
@@ -1769,7 +1770,7 @@ class AgnosticCursor(AgnosticBaseCursor):
     comment = MotorCursorChainingMethod()
     allow_disk_use = MotorCursorChainingMethod()
 
-    _Cursor__die = AsyncRead()
+    _die_lock = AsyncRead()
```

Comment:
Do we even need _die_lock defined anymore?

Reply:
Just in `_LatentCursor` it seems.  I removed the others.

Reply:
Yeah the _LatentCursor behavior is odd. That's another reason I want to avoid implementing this pattern in pymongo.